### PR TITLE
Save page

### PR DESCRIPTION
**Pull request by gwincr11**
Motivation:
  - Create the ability to save a page much like we save a screenshot.
    relates to HashNuke/hound/issues/131
Changes:
  - Add a save page helper.
  - Add tests.

### CODE REVIEW

File: test/helpers/save_page_test.exs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+defmodule SavePageTest do
+  use ExUnit.Case
+  use Hound.Helpers
+
+  hound_session
+
+  test "should save the page" do
+    navigate_to("http://localhost:9090/page1.html")
+    path = save_page("screenshot-test.png")
```
Comment (tuvistavie) [inline test/helpers/save_page_test.exs]: This should end in .html

**Comment by tuvistavie** [comment]
Hi, thank you very much.
I just added a little comment and I think we can merge!

**Comment by gwincr11** [comment]
Oh, geez.... Good catch, just fixed this and pushed it up.

**Comment by tuvistavie** [comment]
Great, thank you!

**Pull request merged**